### DISCUSSION TITLE
On a single machine with 16 cores and 10 hard disks  how to config

### DISCUSSION THREAD

**Discussion by zengyaowu** [minio/minio#19345] [Q&A] [answered]
On a single machine with 16 cores and 10 hard disks, I want to use MINIO as an object storage solution. How can I configure the system such that when uploading an object for storage, it organizes files and directories according to their original structure, ensuring that a single file is only stored on one hard disk without being split into many pieces due to erasure coding? I am willing to tolerate the loss of data from individual failing disks and can recover the data from elsewhere. What configuration should I apply to achieve this setup?

**Comment by klauspost** [comment, marked as answer]
All modes will write metadata and data separately - even if you combine your 10 disks in a software RAID. There is no direct file mapping available.

**Reply by zengyaowu** [reply]
thanks!
